cdptools/databases/document_store_database.py

```diff
@@ -0,0 +1,840 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from abc import ABC, abstractmethod
+from datetime import datetime
+from typing import Any, Dict, List, NamedTuple, Optional, Tuple, Union
+
+import pandas as pd
+
+from . import exceptions
+
+###############################################################################
+
+
+cdp_tables = [
+    "event",
+    "person",
+    "body",
+    "file",
+    "trasncript",
+    "seat",
+    "role",
+    "minutes_item",
+    "event_minutes_item",
+    "vote",
+    "matter",
+    "matter_type",
+]
```

Comment:
Thinking about this a bit more. It may be nice to make this an enum class so people can do `CDPTables.event` or `CDPTables.transcript`. Property access better than random strings as strings can have typos and properties are solved by IDEs usually. I see `transcript` is typoed in here for example.

```python
class CDPTables(Enum):
    event = "event"
    person = "person"
    ...
```

Because it inherits `Enum` people can still loop over them if they want.

Reply:
Sounds good, will do